src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -204,15 +199,13 @@ public BitArray(BitArray bits)
         ** Exceptions: ArgumentOutOfRangeException if index < 0 or
         **             index >= GetLength().
         =========================================================================*/
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool Get(int index)
         {
-            if ((uint)index >= (uint)Length)
-            {
-                throw new ArgumentOutOfRangeException(nameof(index), index, SR.ArgumentOutOfRange_Index);
-            }
+            if ((uint)index >= (uint)m_length)
+                ThrowArgumentOutOfRangeException(index);
 
-            int elementIndex = Div32Rem(index, out int extraBits);
```

Comment:
Is `Div32Rem` still used elsewhere? Would it be beneficial to just use `Math.DivRem` instead in those remaining places?

Reply:
@tannergooding 
It is used. I haven't changed all the places as wanted  to improve specifically Get and Set.